docs/dev-overview/transaction-receipts.md

```diff
@@ -1,19 +1,15 @@
 # Transaction receipts (for recipient)
 
-### User experience
+### Motivation
 
-Similar to checking [transaction statuses](check-transaction-status.md), users can check the status of sent payments by looking up their balance history in their app's _Settings_:
-
-![Locate the balance history in the app's Settings.](../images/balance-history.jpeg) ![Users can check the status of sent transaction statuses in their balance history.](../images/transaction-receipts.jpeg)
+Similar to checking [transaction statuses](check-transaction-status.md), users of a MobileCoin wallet
+application will need to check the status of payments received.
```

Comment:
Please explicitly specify that Sender and Recipient have different methods of checking the transaction status

Reply:
Done